style.css

```diff
@@ -0,0 +1,148 @@
+* {
+  margin: 1%;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+/* Header with Logo & Navbar */
+.header {
+  display: flex;
```

Comment:
<img width="780" alt="1" src="https://github.com/HossainAraf/Portfolio-Mobile-First/assets/77843050/3a05404a-13dc-4fb6-96df-5a4db1c66218">

- [ ] As highlighted in red, let me explain how each part for a better understanding, remember, you need to match the figma template as closely as you can, see the header design, your hamburger is in red and not sharing the same design as the original template, and again, your header as a small background that should not be present in the design, please modify them on the go.
- [ ] About your text, the hero's main title is smaller than the original one.
- [ ] The paragraph color is more black than the original one, I think modifying the color to match the figma template is what is required here.
- [ ] the `let's connect` text should be in capital letters
- [ ] Your social media icons are not well-spaced, please update your padding to make the content breath abit
- [ ] The last one is your background image, please position it as you can see it on the original design on the left